templates/init_migration.sh.template

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+cd "$(dirname "$0")"
```

Comment:
could be worth a comment, I don't know without looking it up what this is doing anyway

Reply:
Pushed fix.